cli/src/org/partiql/cli/main.kt

```diff
@@ -109,6 +109,8 @@ private val inputFormatOpt = optParser.acceptsAll(listOf("input-format", "if"),
 private val wrapIonOpt = optParser.acceptsAll(listOf("wrap-ion", "w"), "wraps Ion input file values in a bag, requires the input format to be ION, requires the query option")
     .availableIf(queryOpt)
 
+private val monochromeOpt = optParser.acceptsAll(listOf("monochrome", "m"), "removes syntax highlighting for the REPL")
+
```

Comment:
What is the motivation behind this? I'm wondering if there's even a need. Maybe if someone has a terminal color scheme with poor contrast? I'm not sure if we need it, but that's just my opinion

Reply:
Yeah, just if a user experiences poor contrast with their theme.